catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswQueryFactory.java

```diff
@@ -232,6 +233,90 @@ private Filter transformCustomFunctionToFilter(Filter filter) {
         return filter;
     }
 
+    private QueryRequest normalizeSort(QueryRequest request) {
+        if (request == null) {
+            return null;
+        }
+
+        Query query = request.getQuery();
+        if (query == null) {
+            return request;
+        }
+
+        List<SortBy> sortBys = new ArrayList<>();
+
+        // Primary sort parameter
+        SortBy sortBy = query.getSortBy();
+        SortBy newSortBy = normalizeSortBy(sortBy);
```

Comment:
[pencil] `newSortBy` doesn't seem like a useful name for the way in which you're using it. Maybe `normSortBy`? Maybe `sortBy` and inline the current `sortBy` variable, i.e. the following?
```
SortBy sortBy = normalizeSortBy(query.getSortBy());
if (sortBy != null) {
    sortBys.add(newSortBy);
}
```

Reply:
I like the inline suggestion